js/renovation/ui/scroll_view/scrollable_native.tsx

```diff
@@ -218,11 +220,12 @@ export class ScrollableNative extends JSXComponent<ScrollableInternalPropsType>(
   }
 
   get cssClasses(): string {
-    const { direction } = this.props;
+    const { direction, classes } = this.props;
 
     const classesMap = {
-      'dx-scrollview dx-scrollable dx-scrollable-native dx-scrollable-renovated dx-scrollable-native-generic': true,
+      'dx-scrollable dx-scrollable-native dx-scrollable-renovated dx-scrollable-native-generic': true,
```

Comment:
cssClasses returns new value and there should be failed tests.
What test was changed?

Reply:
The scrollable widget doesn't know anything about outer scrollView widget. I transfered the logic add 'dx-scrollview' class into scrollView component.

Reply:
ok.
And let's add a new test to check that ScrollView generates DOM element with the dx-scrollview class

Reply:
https://github.com/DevExpress/DevExtreme/pull/15548/files#diff-83a08aabde71752a10f9af9f159152efb32f5a880f731938482881866c01c583R11 - we added it